lib/modules/spoilerTags.js

```diff
@@ -0,0 +1,28 @@
+modules['spoilerTags'] = {
+	moduleID: 'spoilerTags',
+	moduleName: 'Global Spoiler Tags',
+	category: 'UI',
+	options: { },
+	description: 'Hides spoiler-tagged comments globally across Reddit',
+	isEnabled: function() {
+		return RESConsole.getModulePrefs(this.moduleID);
+	},
+	include: [
+		/^https?:\/\/([a-z]+)\.reddit\.com\/user\/.*/i
```

Comment:
Yes? [`profileRegex`](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/a2acbcd09e7293c0fc548eae8d462184912289c5/lib/core/utils.js#L287) should match all URLs that this line does.